navigation/waypoint_manager/scripts/waypoint_manager.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+import rclpy
+from rclpy.node import Node
+from vortex_msgs.srv import Waypoint
+from nav_msgs.msg import Path
+from geometry_msgs.msg import PoseStamped, Point
+
+
+class WaypointManager(Node):
+    """
+    Nodes created:
+        WaypointManager
+    Subscribes to:
+    Publishes to:
+    """
+    
+    def __init__(self):
+        """
+        Initializes WaypointManager class.
+
+        Initializes node, sets up action client, services, and publishers.
+        """
+        super().__init__("WaypointManager")
+
+        self.waypoint_list = []
+
+        # Service for getting the list of waypoints and client for sending the list of waypoint
+        self.waypoint_service = self.create_service(Waypoint, 'waypoint', self.waypoint_callback)
+        self.waypoint_list_client = self.create_client(Waypoint, 'waypoint_list')
+
+        # Services
+        self.get_logger().info('kommet til services')
+        self.add_waypoint_service = self.create_service(Waypoint, 'add_waypoint', self.add_waypoint_callback)
+        self.get_logger().info('kommet til etter at man har laget add_waypoint_services')
+
+        # nav_msgs Path to visualize LOS in Rviz
+        self.path_pub = self.create_publisher(Path, 'los_path', 10)
+        self.path = Path()
+        self.path.header.frame_id = 'world'
```

Comment:
What is the motivation behind this?

Reply:
self.path_pub was left on accident. it originally was a publisher for LosPathFollowing. Based on what i have read up on (please correct me if i am wrong) the meaning of path is that it represent a sequence of poses (positions and orientations) that make up a path for a robot to follow, but i am actually unsure if it is used in the script

Reply:
For debugging/visualization it can be really helpful to have this, not sure where it should live though!